src/features/classification/ClassifiedBadge.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../styles/variables';
+@import './mixins';
+
+.bdl-ClassifiedBadge,
+.bdl-ClassifiedBadge:hover {
```

Comment:
Can this live as `&:hover`, as the other one does? Is `&:focus` needed?

Reply:
I think I could but it was just the same style since this case badge doesn't have a hover state, same as non-hovered state. The badge ends up in breadcrumbs that have hover state, so this is just overriding it. The badge till be in a button, so the parent has its focus and hover.